docs/installation.rst

```diff
@@ -69,6 +69,12 @@ Installation
 
     If not set, django-newsletter will fall back to Django's default TextField
     widget.
+#)  Configure delay between each Email (optional).
+
+    As default the configuration delay between each Email is 0 but you can easily
+    change it to what ever you feel is good for your company::
+
+        DELAY_BETWEEN_EACH_EMAIL = 0
 
```

Comment:
Please make sure here the number is seconds and can be factional (for 10000 emails waiting 0.1 or 1 second makes a huge difference. Also, if you can, please write Email as email.